README.md

```diff
@@ -4,6 +4,47 @@
 
 Next SEO is a plug in that makes managing your SEO easier in Next.js projects and is compatible with version`6.0.0`+ of Next.js.
 
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+<!-- https://github.com/thlorenz/doctoc -->
```

Comment:
Do you think it would be worth adding this as a script to the `package.json` so the contents could be updated each time the docs are updated?

Reply:
I have updated it and inluded it in the pre-commit hook